### PR TITLE
Fix FIAT<->CRYPTO bug and keep CRYPTO<->CRYPTO working

### PR DESCRIPTION
**Pull request by j7p2**
Fix Open Issue #7 
It should be:
String baseCurrencyCode = currencyCode;
String counterCurrencyCode = isCryptoCurrency ? Res.getBaseCurrencyCode() : currencyCode;

Instead of:
String baseCurrencyCode = isCryptoCurrency ? currencyCode : Res.getBaseCurrencyCode();
String counterCurrencyCode = isCryptoCurrency ? Res.getBaseCurrencyCode() : currencyCode;

### CODE REVIEW

File: core/src/main/java/bisq/core/offer/CreateOfferService.java [hunk 1]
```diff
@@ -62,293 +62,213 @@
...
-        // MinBuyerSecurityDepositAsCoin from Restrictions.
-        return Coin.valueOf(Math.max(Restrictions.getMinBuyerSecurityDepositAsCoin().value, value.value));
-    }
-
-    private Coin getBoundedSellerSecurityDeposit(Coin value) {
-        // We need to ensure that for small amount values we don't get a too low BTC amount. We limit it with using the
-        // MinSellerSecurityDepositAsCoin from Restrictions.
-        return Coin.valueOf(Math.max(Restrictions.getMinSellerSecurityDepositAsCoin().value, value.value));
-    }
+	private final TxFeeEstimationService txFeeEstimationService;
```
Comment (woodser) [inline core/src/main/java/bisq/core/offer/CreateOfferService.java:65]: Whitespace should be preserved as spaces instead of tabs.

**Comment by j7p2** [comment]
Sorry for modifying the file a lot. But there were too many white spaces and blank lines. My plugin to code in Java tries to organize the code in a the best way so it has removed all the unnecessary spaces to keep formatting according Java conventions.
Should I redo the PR with only the changed line?

**Comment by woodser** [comment]
> Should I redo the PR with only the changed line?

Yes please.  Your IDE should let you configure how to handle whitespace.

The PR should also be rebased on master so it merges clean.

**Comment by j7p2** [comment]
Ok ;)

**Pull request closed without merging**